src/lib/classes/lsp8-identifiable-digtal-asset.spec.ts

```diff
@@ -46,4 +47,43 @@ describe('LSP8IdentifiableDigitalAsset', () => {
     const ownerAddress = await LSP8IdentifiableDigitalAsset.owner();
     expect(ownerAddress).toEqual('0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266');
   });
+
+  it('should deploy reactive', (done) => {
+    const myLSPFactory = new LSPFactory(provider, signer);
+
+    const lsp8IdentifiableDigitalAsset$ = myLSPFactory.LSP8IdentifiableDigitalAsset.deploy(
+      {
+        controllerAddress: signer.address,
+        name: 'TOKEN',
+        symbol: 'TKN',
+      },
+      {
+        libAddress: baseContract.address,
+        deployReactive: true,
+      }
+    ) as Observable<DeploymentEvent>;
+
+    let lsp8Address: string;
+
+    lsp8IdentifiableDigitalAsset$.subscribe({
+      next: (deploymentEvent: DeploymentEvent) => {
+        if (
+          deploymentEvent.receipt?.contractAddress &&
+          deploymentEvent.contractName === 'LSP8IdentifiableDigitalAsset'
+        ) {
+          lsp8Address = deploymentEvent.receipt.contractAddress;
+        }
+      },
+      error: () => {
+        done();
+      },
+      complete: async () => {
+        const lsp7DigitalAsset = LSP8Mintable__factory.connect(lsp8Address, signer);
+
+        const ownerAddress = await lsp7DigitalAsset.owner();
+        expect(ownerAddress).toEqual(signer.address);
+        done();
```

Comment:
this looks very very similar to the test of the lsp7 - maybe you want to extract this logic into a helper function or smtgh so you don't duplicate / copy/paste the exact same test?

You can tell it is a copy paste because the name of the variables still mention `lsp7`....

```
lsp7DigitalAsset
```

To solve this you can give a generic name `address`, `factory`, etc. instead of `lsp8Address`....